django/views/generic/edit.py

```diff
@@ -140,6 +140,12 @@ def get_form_kwargs(self):
             kwargs.update({'instance': self.object})
         return kwargs
 
+    def get_model(self):
+        """
+        Returns the model of the form_class.
+        """
+        return self.get_form_class().Meta.model
```

Comment:
I believe `super(ModelFormMixin, self).get_model()` should **always** have priority over `self.get_form_class().Meta.model`.